cmd/build.go

```diff
@@ -297,10 +343,6 @@ func (c buildConfig) Configure(f fn.Function) fn.Function {
 // Skipped if not in an interactive terminal (non-TTY), or if --confirm false (agree to
 // all prompts) was set (default).
 func (c buildConfig) Prompt() (buildConfig, error) {
-	if !interactiveTerminal() {
-		return c, nil
-	}
```

Comment:
why are we removing the check for non interactive terminal? The function itself is called `Prompt()`. If theres functionality that should be performed regardless of interactivity of the terminal, I think it should be done elsewhere. Either it must be moved prior the function call or the function must change.

I see that a check is done later but the function has now become more of a validity function instead which im not sure is the right approach. WDYT @lkingland ?